React-Template-master/src/pages/ForgotPassword.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Typography, Grid, TextField, Button } from '@mui/material';
+import React, { useState } from 'react';
+
+export const ForgotPassword = (): JSX.Element => {
+    const [email, setEmail] = useState('');
+
+    return (
+        <Box
+            sx={{
+                marginTop: '83px',
+                display: 'flex',
+                flexDirection: 'column',
+                alignItems: 'left',
+                textAlign: 'left'
+            }}>
+            <Box sx={{ height: '130px', mt: '77px' }}>
+                <Typography
```

Comment:
Override the typography in theme instead of repeating the styled.

Reply:
okay